app/src/main/kotlin/nl/eduid/screens/verifyidentity/VerifyIdentityScreen.kt

```diff
@@ -94,23 +96,26 @@ fun VerifyIdentityScreen(
             val intent = Intent(Intent.ACTION_VIEW, Uri.parse(it))
             launcher.launch(intent)
         },
+        expandMoreOptions = {
+            viewModel.expandMoreOptions()
+        },
         padding = padding
     )
 }
 
 @Composable
 fun VerifyIdentityScreenContent(
     isLoading: Boolean,
+    moreOptionsExpanded: Boolean,
     errorData: ErrorData?,
     dismissError: () -> Unit,
     goToBankSelectionScreen: () -> Unit,
     requestInstitutionLink: () -> Unit,
     requestEidasLink: () -> Unit,
+    expandMoreOptions: () -> Unit,
     openSupportUrl: (String) -> Unit,
     padding: PaddingValues = PaddingValues(),
 ) {
-
```

Comment:
When coming back to the screen, it would collapse again, because the UI was recreated from scratch.
By putting it in the VM, it would remember the previous value and would stay expanded